diff --git a/node_modules/qrcodejs2/qrcode.js b/node_modules/qrcodejs2/qrcode.js
index 9f7082b..89c7b2d 100644
--- a/node_modules/qrcodejs2/qrcode.js
+++ b/node_modules/qrcodejs2/qrcode.js
@@ -291,7 +291,8 @@ var QRCode;
 
 		// Android 2.1 bug workaround
 		// http://code.google.com/p/android/issues/detail?id=5141
-		if (this._android && this._android <= 2.1) {
+		// if (this._android && this._android <= 2.1) {
+			if (this && this._android && this._android <= 2.1) {
 	    	var factor = 1 / window.devicePixelRatio;
 	        var drawImage = CanvasRenderingContext2D.prototype.drawImage;
 	    	CanvasRenderingContext2D.prototype.drawImage = function (image, sx, sy, sw, sh, dx, dy, dw, dh) {
